Changes to "m3tohtml" to use the new "m3configvars" package.

===================================================================
RCS file: /home/jdp/m3-cvs/m3/m3tohtml/src/Main.m3,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 Main.m3
--- m3/m3tohtml/src/Main.m3.orig	1996/09/24 05:22:00	1.1.1.1
+++ m3/m3tohtml/src/Main.m3	1996/09/24 05:32:40
@@ -7,7 +7,7 @@
 MODULE Main;
 
 IMPORT Text, Rd, Wr, Stdio, Thread, Fmt, Time;
-IMPORT OSError, FileRd, FileWr, Pathname, FS, M3Config;
+IMPORT OSError, FileRd, FileWr, Pathname, FS, M3ConfigVars;
 IMPORT MarkUp, M3DB, HTMLDir, FilePath, Process;
 <*FATAL Thread.Alerted*>
 
@@ -31,8 +31,8 @@
     WHILE NOT Rd.EOF (rd) DO
       file := Rd.GetLine (rd);
       IF Text.GetChar (file, 0) = '$' THEN
-        pkg := Text.Sub (file, 1) & M3Config.PATH_SEP;
-        proj_pkg := M3Config.PKG_USE & M3Config.PATH_SEP & pkg;
+        pkg := Text.Sub (file, 1) & M3ConfigVars.PATH_SEP;
+        proj_pkg := M3ConfigVars.PKG_USE & M3ConfigVars.PATH_SEP & pkg;
       ELSE
         INC (n_sources);
         sources := NEW (Source, next := sources,
@@ -53,7 +53,7 @@
     sources := b;
   END ReadFileList;
 
-VAR(*CONST*) Build_dir_len := Text.Length (M3Config.BUILD_DIR);
+VAR(*CONST*) Build_dir_len := Text.Length (M3ConfigVars.BUILD_DIR);
 
 PROCEDURE FixDerived (filename: TEXT): TEXT =
   VAR i: INTEGER;
@@ -64,13 +64,14 @@
 
     i := 0;
     WHILE (i < Build_dir_len) DO
-      IF Text.GetChar (filename, i) # Text.GetChar (M3Config.BUILD_DIR, i) THEN
+      IF Text.GetChar (filename, i) # Text.GetChar (M3ConfigVars.BUILD_DIR, i)
+      THEN
         RETURN filename;
       END;
       INC (i);
     END;
 
-    IF Text.GetChar (filename, i) = Text.GetChar (M3Config.PATH_SEP, 0) THEN
+    IF Text.GetChar (filename, i) = Text.GetChar (M3ConfigVars.PATH_SEP, 0) THEN
       filename := "derived" & Text.Sub (filename, i);
     END;
     RETURN filename;
===================================================================
RCS file: /home/jdp/m3-cvs/m3/m3tohtml/src/m3makefile,v
retrieving revision 1.1.1.2
diff -u -r1.1.1.2 m3makefile
--- m3/m3tohtml/src/m3makefile.orig	1996/09/24 05:29:00	1.1.1.2
+++ m3/m3tohtml/src/m3makefile	1996/09/24 05:35:10
@@ -10,6 +10,7 @@
 % m3_option ("-Y0@/udir/kalsow/pkg/m3/compiler/DS/m3c@-tDS3100@")
 % override (m3tools, "/udir/kalsow/pkgs")
 
+import ("m3configvars")
 import ("libm3")
 import ("m3tools")
 
